server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemReader_7_2.java

```diff
@@ -44,6 +44,7 @@
  *
  */
 class EndpointSubsystemReader_7_2 implements XMLStreamConstants, XMLElementReader<List<ModelNode>> {
+   public static final XMLElementReader<List<ModelNode>> INSTANCE = new EndpointSubsystemReader_7_2();
```

Comment:
I noticed these new INSTANCE fields all over?  What are they used for?  I don't see any references.